cmd/pbstatuser/main.go

```diff
@@ -0,0 +1,118 @@
+
+	// Beware there is no database connection, no job queue, most of what api/worker processes have
+	// is not available because it should not be needed. The worker will simply use SDKs to check
+	// statuses and send out results via bus.Send function to Sources.
+	// Also make sure that all goroutines are closed correctly on context cancel. A WaitGroup
+	// would be a really good here: https://gist.github.com/fracasula/b579d52daf15426e58aa133d0340ccb0
+}
+
+func main() {
+	ctx := context.Background()
+	config.Initialize("config/api.env", "config/statuser.env")
+
+	// initialize stdout logging and AWS clients first
+	logging.InitializeStdout()
+	cloudwatchlogs.Initialize()
+
+	// initialize cloudwatch using the AWS clients
+	logger, clsFunc, err := logging.InitializeCloudwatch(log.Logger)
+	if err != nil {
+		log.Fatal().Err(err).Msg("Error initializing cloudwatch")
+	}
+	defer clsFunc()
+	log.Logger = logger
+	logging.DumpConfigForDevelopment()
+
+	// setup structured logging
+	hostname, err := os.Hostname()
+	if err != nil {
+		hostname = "unknown-hostname"
+	}
+	logger = logger.With().
+		Timestamp().
+		Str("hostname", hostname).
+		Bool("statuser", true).
+		Logger()
+
+	// initialize telemetry
+	tel := telemetry.Initialize(&log.Logger)
+	defer tel.Close(ctx)
+
+	// initialize platform bus
+	log.Info().Msg("Initializing platform bus")
+	err = bus.InitializeKafkaBroker()
+	if err != nil {
+		logger.Fatal().Err(err).Msg("Unable to initialize the platform bus")
+	}
+
+	// start the consumer
+	cancelCtx, consumerCancelFunc := context.WithCancel(ctx)
+	go bus.Consume(cancelCtx, bus.AvailabilityStatusRequestTopic, processMessage)
+
+	// metrics
+	log.Info().Msgf("Starting new instance on port %d with prometheus on %d", config.Application.Port, config.Prometheus.Port)
+	metricsRouter := chi.NewRouter()
+	metricsRouter.Handle(config.Prometheus.Path, promhttp.Handler())
+	metricsServer := http.Server{
+		Addr:    fmt.Sprintf(":%d", config.Prometheus.Port),
+		Handler: metricsRouter,
+	}
+
+	waitForSignal := make(chan struct{})
+	go func() {
+		sigint := make(chan os.Signal, 1)
+		signal.Notify(sigint, os.Interrupt, syscall.SIGTERM)
+		<-sigint
+		if err := metricsServer.Shutdown(context.Background()); err != nil {
+			log.Fatal().Err(err).Msg("Metrics service shutdown error")
+		}
+		close(waitForSignal)
+	}()
+
+	go func() {
+		if err := metricsServer.ListenAndServe(); err != nil {
+			var errInUse syscall.Errno
+			if errors.As(err, &errInUse) && errInUse == syscall.EADDRINUSE {
+				log.Warn().Err(err).Msg("Not starting metrics service, port already in use")
+			} else if !errors.Is(err, http.ErrServerClosed) {
+				log.Warn().Err(err).Msg("Metrics service listen error")
+			}
+		}
+	}()
+
+	log.Info().Msg("Worker started")
+	<-waitForSignal
+	log.Info().Msg("Consumer shutdown initiated")
+	consumerCancelFunc()
+	log.Info().Msg("Shutdown finished, exiting")
+}
```

Comment:
The statuser process is really simple - no database, no job queue, just kafka connection and prometheus.

Which reminds me, we should add some telemetry, but in another PR. Kafka exposes statistics via `Stats` function.